charts/factorio-server-charts/Chart.yaml

```diff
@@ -20,7 +20,7 @@ sources:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
+version: 2.1.1
```

Comment:
This should be 2.2.0 since it's new functionality added to the chart in a backwards compatible way.